main.nf

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env nextflow
 
-nextflow.preview.dsl = 2
+nextflow.enable.dsl=2
```

Comment:
when did they add this one? We're currently using v20.10.0.5430 -- if it's beyond that, we should add something to the README

Reply:
Not sure, but I'm on 22.04.3 and getting this error:
```{bash}
N E X T F L O W  ~  version 22.04.3
Preview nextflow mode 'preview' is not supported anymore -- Please use `nextflow.enable.dsl=2` instead
```